src/data_gradients/feature_extractors/segmentation/bounding_boxes_area.py

```diff
@@ -55,7 +68,9 @@ def aggregate(self) -> Feature:
             bandwidth=0.4,
             tight_layout=True,
         )
-        json = dict(train=dict(df[df["split"] == "train"]["bbox_area"].describe()), val=dict(df[df["split"] == "val"]["bbox_area"].describe()))
+        json = dict(
```

Comment:
Hard-coded split names here. Please change to iteration over `df["split"].unique()`

Reply:
Done here [00faa28](https://github.com/Deci-AI/data-gradients/pull/129/commits/00faa28b39abe2bdb072b6712015ce3ec006ff72)